FUNDABLES.md

```diff
@@ -54,6 +54,33 @@ frontend development, testing, DevOps/infrastructure/platform services,
 user experience work, technical writing for end users, project
 management, and community outreach.
 
+#### Port auditwheel to Windows and Mac
+
+The packaging tool [auditwheel](https://github.com/pypa/auditwheel)
+"is a command line tool to facilitate the creation of Python wheel
+packages for Linux (containing pre-compiled binary extensions) that
+are compatible with a wide variety of Linux distributions" and key
+standards. It can inspect a wheel, checking whether it is
+standards-compliant. It can also repair a wheel. If a wheel depends on
+libraries that are not on the system, it can rewrite that wheel and inject
+libraries needed, parsing and rewriting ELF data. It can also repair the
+relevant `manylinux` tag(s) on a wheel.
+
+However, no such utility exists on Windows and Mac OS X, and so
+package maintainers on those platforms face trouble creating and
+debugging their packages. [Developers would like to add Windows and
+Mac support to
+auditwheel.](https://discuss.python.org/t/need-auditwheel-like-utility-for-other-platforms/2028/4)
+This will especially be useful for scientific programmers, since they
+often create Python applications or libraries that include binaries
+written in other languages, and wheel distributions of those packages
+are prone to complication. This will also smooth the way for a
+potential generic wheel-building service.
```

Comment:
I think the generic wheel-building service is somewhat orthogonal... maybe a better way to put it is, having a simpler and more consistent workflow here makes it easier for package maintainers to use generic off-the-shelf automation? Which means that whatever automation is available (Github Actions, Travis, a hypothetical PyPI wheel building service) will be usable by more maintainers.